### DISCUSSION TITLE
Profile Management

### DISCUSSION THREAD

**Discussion by noel-vargas** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#135] [Q&A]
Hi, I currently have this working demo for the profile management tab. would love to have some feedback. 
https://github.com/user-attachments/assets/b299e657-5514-43a2-a165-c5a895963c77

**Comment by YanehRuiz** [comment]
Hey! I really like what you've done with the profile editing section. I think adding a 'Save' button would be super helpful to make sure any changes are saved properly. Also, I noticed that the 'Sign Out' button might be a bit confusing where it is right now—maybe moving it to a different spot could help users find it more easily. Overall, great job! I can’t wait to see what you come up with next!
